README.md

```diff
@@ -51,6 +51,8 @@ and move each window to its workspace after session restore.
 
 [amo]: https://addons.mozilla.org/en-US/firefox/addon/i3-workspaces/
 
+[For Gentoo you can emerge from Guru](https://github.com/gentoo/guru/tree/master/x11-wm/firefox-i3-workspaces).
```

Comment:
Let’s attribute that to you, like

```
<your preferred name form> maintains a Gentoo package in Guru: [x11-wm/firefox-i3-workspaces](…)
```

(<your preferred name form> could also be the github @.)

Also, please see the two questions I left [there](https://github.com/gentoo/guru/commit/d215ee8db96e85b1cf14a220a0eaf165a63406ce).